modules/sdk-lib-mpc/src/tss/ecdsa-dkls/dsg.ts

```diff
@@ -65,6 +99,8 @@ export class Dsg {
     try {
       const payload = this.dsgSession.createFirstMessage().payload;
       this._deserializeState();
+      this.dsgSessionBytes = this.dsgSession.toBytes();
+      this.dsgSession = undefined;
```

Comment:
adding a similar behaviour as the finally code block in `handleIncomingMessages`